rollbar-java/src/main/java/com/rollbar/notifier/util/ObjectsUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.rollbar.notifier.util;
```

Comment:
Could you please provide some tests for these methods? Also, I think that the indentation level is not correct.